src/components/regions/ProbeDetails.svelte

```diff
@@ -49,7 +48,7 @@ function probeIsSelected(probeToTest) {
 </style>
 
 <div class="drawer details-drawer">
-{#if !$telemetrySearch.loaded}
+{#if !$store.probe.loaded}
```

Comment:
@spasovski would be good to go through this file and make sure we're pulling in alll the relevant metadata out of `$store.probe`. Could you make sure this file reaches parity w/ whatever was in $probe before?

Reply:
Yep I already had that in the initial PR should be easy.